plugins/audit_logs/girder_audit_logs/cleanup.py

```diff
@@ -0,0 +1,26 @@
+"""
+This script is for deleting old audit log entries from the database. Example invocation:
+
+    girder audit_logs_cleanup --days=30 --types=rest.request
+"""
+
+import click
+import datetime
+from girder_audit_logs import Record
+
+
+@click.command(name='audit_logs_cleanup')
+@click.option('--days', type=click.INT, help='How many days to preserve records', default=90)
+@click.option('--types', help='Which record types to remove as a comma separated list. If not '
+              'provided, removes all record types.')
+def cleanup(days, types):
+    filter = {'when': {'$lt': datetime.datetime.utcnow() - datetime.timedelta(days=days)}}
+
+    if types:
+        filter['type'] = {'$in': types.split(',')}
+
+    print('Deleted %d log entries.' % Record().collection.delete_many(filter).deleted_count)
```

Comment:
Nitpick, maybe use `click.echo` http://click.palletsprojects.com/en/7.x/utils/#printing-to-stdout.

Reply:
I love this part of the documentation for that function:

> and it will never fail (except in Python 3